data/bunrodea/bunrodea missions.txt

```diff
@@ -0,0 +1,217 @@
+			`	You pick up the device and look at it. It springs open in your hand, causing you to drop it. After it falls to the ground, it begins projecting a holographic star map. You recognize some of the systems on the map as those where you fought an alien fleet. In fact, the alien ship in front of you looks similar to those you fought. A single system within that region is highlighted. The aliens return to their ship, which becomes invisible again. Although you can't see it, you hear it take off and fly away.`
+				accept
+			label dodge
+			`	You dodge for cover behind your ship. After a few seconds, you hear a springing sound. When you look back out from behind your ship, the aliens are gone, and the device they threw at you is now projecting a holographic star map. You recognize some of the systems on the map as those where you fought an alien fleet. In fact, the alien ship in front of you looks similar to those you fought. A single system within that region is highlighted. The alien ship becomes invisible once again. Although you can't see it, you hear it take off and fly away.`
+				accept
+	
+	on accept
+		"reputation: Bunrodea" = 0
+		"reputation: Bunrodea (Guard)" = 0
+		"reputation: Bunrodea (Megasa)" = 0
+		"reputation: Bunrodea (Erabu)" = 0
+	
+	on complete
+		"reputation: Bunrodea" = 10
+		conversation
+			`As you enter the atmosphere of the alien planet, you're flanked by two large alien ships which guide you to the surface, leading you to a particular landing zone. When you leave your ship, you're met by two aliens that look similar to the ones that startled you on <origin>, including having the same weaponry.`
+			`	As you take a moment to look around, you find yourself in the middle of a massive alien metropolis. Skyscrapers taller than anything you've ever seen in human space cover the horizon, although most of the buildings around you appear far more elegant in architecture. Directly ahead is a large palace-like building with a considerable number of steps leading up to the front doors.`
+			`	You take step forward, but one of the aliens says something while waving its weapon around, and you take the hint that you shouldn't be moving right now. The other says something into a communication device, then sits down on the ground and gestures for you to do the same. You suspect that you might be waiting here for a while.`
+			choice
+				`	(Sit down.)`
+					goto sit
+				`	(Turn around and reenter my ship.)`
+			`	You reenter your ship, and the aliens don't move to stop you. You decide to turn your ship back on and attempt to take off, but just as you try to activate the repulsor engines, you hear a loud buzzing sound coming from outside. You look out the nearest window, and see that the aliens have moved some sort of vehicle near your ship which is buzzing with energy. You attempt to take off, but your ships' repulsor engines aren't moving you anywhere. The vehicle seems like it is deactivating any nearby repulsors.`
```

Comment:
>Maybe it would be better to refer to them as engines instead of repulsor engines.

Except a ship's repulsor engines are distinct from the normal engines that you install.